app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ErrorsController < ApplicationController
+  def not_found
+    render(:status => 404)
```

Comment:
Have a look at codeclimate for the new syntax (guess it's something like `render status: 404` ...)